### PR TITLE
Update proceedings.md

### PR DESCRIPTION
**Pull request by cbcunc**
Add 2022 proceedings.

### CODE REVIEW

File: content/pages/proceedings.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ Save_as: proceedings/index.html
 **ISSN: 2575-9752**
 **[https://doi.org/10.25080/issn.2575-9752](https://doi.org/10.25080/issn.2575-9752)**
 
+**[SciPy 2022](https://conference.scipy.org/proceedings/scipy2022)**
+ *21th Python in Science Conference - Austin, Texas (July 11 - 17, 2022)*
```
Comment (stefanv) [inline content/pages/proceedings.md:9]:
```suggestion
 *21st Python in Science Conference - Austin, Texas (July 11 - 17, 2022)*
```

**Comment by deniederhut** [comment]
I'm not actually sure if we're regenerating the public site from this static generator anymore? I think the in past Didrik and I have just been editing the html by hand

**Comment by stefanv** [comment]
> I'm not actually sure if we're regenerating the public site from this static generator anymore? I think the in past Didrik and I have just been editing the html by hand

However it is done, we should ensure that there is only one Source of Truth.

**Comment by achabotl** [comment]
This may not be necessary with the new proceedings site, but merging this anyway for now to clean things up.

**Pull request merged**